server/decoders/account.ts

```diff
@@ -0,0 +1,28 @@
+// ---------------------------------------------------------------------------------------------------------------------
+// Account Decoders
+// ---------------------------------------------------------------------------------------------------------------------
+
+import { array, email, inexact, string, object } from 'decoders';
+import { jsonArrayString, jsonObjectString, stringWithLength, withDefault } from './utils';
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+export const accountRecDecoder = object({
+    id: string,
+    email,
+    name: stringWithLength(3, 255),
+    avatar: stringWithLength(3, 255),
+    permissions: jsonArrayString(string),
+    settings: jsonObjectString(inexact({}))
```

Comment:
Someday I hope we can move away from inexact, but for now it's fine.

Reply:
I have a todo for that in the Account model. So I'm right there with you.